### DISCUSSION TITLE
How do  solve problems when downloading model weights

### DISCUSSION THREAD

**Discussion by chris011225** [mikel-brostrom/yolo_tracking#1053] [Q&A]
I'm a green hand.When executing the following code：python examples/track.py --yolo-model yolov8n --source vid.mp4，report an error：requests.exceptions.SSLError: HTTPSConnectionPool(host='drive.google.com', port=443): Max retries exceeded with url: /uc?id=12uD5FeVqLg9-AFDju2L7SQxjmPb4zpBN (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1122)')))，vid.mp4 is in yolo_tracking folder

**Comment by mikel-brostrom** [comment]
https://github.com/mikel-brostrom/yolo_tracking/issues/850

**Reply by chris011225** [reply]
Thank you very much sir

**Comment by chris011225** [comment]
Excuse me，sir，i am the green hand who ask you question on GitHub，i still can not download the weight document named mobilenetv2_x1_4_dukemtmcreid.pt，i will appreciate it if you emailed me the weight document.Thank god let me meet you.




1.27
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "mikel-brostrom/yolo_tracking"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年8月3日(星期四) 晚上11:06
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [mikel-brostrom/yolo_tracking] How do  solve problems when downloading model weights (Discussion #1053)





 
Closed #1053 as resolved.
